### ISSUE TITLE
DGS10 data-point very stale

### ISSUE THREAD

**Issue by McQgit** [iexg/IEX-API#1011]
**Clearly describe the data problem**
Using the data-points api for the 10Y US Treasury gives very stale value of 2.49

**Data URL (The URL you used to access the data without your token):**
https://cloud.iexapis.com/stable/data-points/market/DGS10?token=(redacted)

**What time did the problem occur (including timezone)?**
10:52am June4 eastern

**What is the expected behavior?**
The US Treasury site says the value was 2.07 for day ending Jun 3 and quoting services for Jun 4 trading day are showing roughly 2.13.  These are very different than the 2.49 returned by the IEX api.
[]https://www.treasury.gov/resource-center/data-chart-center/interest-rates/Pages/TextView.aspx?data=yield

**What are the steps to reproduce**
Use iex api as I describe to see value, compare to US Treasury site or other credible quote service

**Your environment (e.g. browser, OS, API version, language):**
n/a

**Additional context**
I spot checked a couple other UST yields and found them similarly stale.  Further, the data-points/market api for metadata returns a current timestamp in its lastUpdated field.

**Comment by evancarter-iex** [comment]
@McQgit is this still an issue? It seems okay on my end.

**Comment by McQgit** [comment]
@evancarter-iex It continued to return 2.49 for at least several days after I posted this but right now it is showing 2.09.  That still seems a day stale as that was the rate ending Jun7 per the US Treasury site I sited.  And, quoting sites today are around 2.15 like yesterday's close.  But, that's not as stale as when I originally reported it.

Note your metadata lastUpdated for that item shows timestamp for end of day Jun11 but 2.09 is not the rate for that closing.  I still think there is a slight problem, in summary.

**Comment by evancarter-iex** [comment]
@McQgit Yeah it looks like the site we get it from will update in a couple of hours with today's data. I'll see if I can make sure it update faster. Thanks
